src/mode/solidity_highlight_rules.js

```diff
@@ -0,0 +1,358 @@
+// rules were taken from https://github.com/raphaelhuefner/ace-mode-solidity
+// all credits to @raphaelhuefner
+
+define(function(require, exports, module) {
```

Comment:
Could you remove the define wrappers, these are not needed anymore.

Reply:
done